### ISSUE TITLE
scipy doc fails to build with numpydoc 0.9.1

### ISSUE THREAD

**Issue by opoplawski** [numpy/numpydoc#223]
Updating numpydoc from 0.8.0 to 0.9.1 appears to have broken the doc build in Fedora:
```
BUILDSTDERR: numpydoc.docscrape.ParseError: multivariate local optimizers is not a item name in 'Find a root of a function in a bracketing interval using Brent\'s method.\n\nUses the classic Brent\'s method to find a zero of the function `f` on\nthe sign changing interval [a , b].  Generally considered the best of the\nrootfinding routines here.  It is a safe version of the secant method that\nuses inverse quadratic extrapolation.  Brent\'s method combines root\nbracketing, interval bisection, and inverse quadratic interpolation.  It is\nsometimes known as the van Wijngaarden-Dekker-Brent method.  Brent (1973)\nclaims convergence is guaranteed for functions computable within [a,b].\n\n[Brent1973]_ provides the classic description of the algorithm.  Another\ndescription can be found in a recent edition of Numerical Recipes, including\n[PressEtal1992]_.  Another description is at\nhttp://mathworld.wolfram.com/BrentsMethod.html.  It should be easy to\nunderstand the algorithm just by reading our code.  Our code diverges a bit\nfrom standard presentations: we choose a different formula for the\nextrapolation step.\n\nParameters\n----------\nf : function\n    Python function returning a number.  The function :math:`f`\n    must be continuous, and :math:`f(a)` and :math:`f(b)` must\n    have opposite signs.\na : scalar\n    One end of the bracketing interval :math:`[a, b]`.\nb : scalar\n    The other end of the bracketing interval :math:`[a, b]`.\nxtol : number, optional\n    The computed root ``x0`` will satisfy ``np.allclose(x, x0,\n    atol=xtol, rtol=rtol)``, where ``x`` is the exact root. The\n    parameter must be nonnegative. For nice functions, Brent\'s\n    method will often satisfy the above condition with ``xtol/2``\n    and ``rtol/2``. [Brent1973]_\nrtol : number, optional\n    The computed root ``x0`` will satisfy ``np.allclose(x, x0,\n    atol=xtol, rtol=rtol)``, where ``x`` is the exact root. The\n    parameter cannot be smaller than its default value of\n    ``4*np.finfo(float).eps``. For nice functions, Brent\'s\n    method will often satisfy the above condition with ``xtol/2``\n    and ``rtol/2``. [Brent1973]_\nmaxiter : int, optional\n    if convergence is not achieved in `maxiter` iterations, an error is\n    raised.  Must be >= 0.\nargs : tuple, optional\n    containing extra arguments for the function `f`.\n    `f` is called by ``apply(f, (x)+args)``.\nfull_output : bool, optional\n    If `full_output` is False, the root is returned.  If `full_output` is\n    True, the return value is ``(x, r)``, where `x` is the root, and `r` is\n    a `RootResults` object.\ndisp : bool, optional\n    If True, raise RuntimeError if the algorithm didn\'t converge.\n    Otherwise the convergence status is recorded in any `RootResults`\n    return object.\n\nReturns\n-------\nx0 : float\n    Zero of `f` between `a` and `b`.\nr : `RootResults` (present if ``full_output = True``)\n    Object containing information about the convergence.  In particular,\n    ``r.converged`` is True if the routine converged.\n\nSee Also\n--------\nmultivariate local optimizers\n  `fmin`, `fmin_powell`, `fmin_cg`, `fmin_bfgs`, `fmin_ncg`\nnonlinear least squares minimizer\n  `leastsq`\nconstrained multivariate optimizers\n  `fmin_l_bfgs_b`, `fmin_tnc`, `fmin_cobyla`\nglobal optimizers\n  `basinhopping`, `brute`, `differential_evolution`\nlocal scalar minimizers\n  `fminbound`, `brent`, `golden`, `bracket`\nn-dimensional root-finding\n  `fsolve`\none-dimensional root-finding\n  `brenth`, `ridder`, `bisect`, `newton`\nscalar fixed-point finder\n  `fixed_point`\n\nNotes\n-----\n`f` must be continuous.  f(a) and f(b) must have opposite signs.\n\nReferences\n----------\n.. [Brent1973]\n   Brent, R. P.,\n   *Algorithms for Minimization Without Derivatives*.\n   Englewood Cliffs, NJ: Prentice-Hall, 1973. Ch. 3-4.\n\n.. [PressEtal1992]\n   Press, W. H.; Flannery, B. P.; Teukolsky, S. A.; and Vetterling, W. T.\n   *Numerical Recipes in FORTRAN: The Art of Scientific Computing*, 2nd ed.\n   Cambridge, England: Cambridge University Press, pp. 352-355, 1992.\n   Section 9.3:  "Van Wijngaarden-Dekker-Brent Method."\n\nExamples\n--------\n>>> def f(x):\n...     return (x**2 - 1)\n\n>>> from scipy import optimize\n\n>>> root = optimize.brentq(f, -2, 0)\n>>> root\n-1.0\n\n>>> root = optimize.brentq(f, 0, 2)\n>>> root\n1.0'
```

**Comment by rgommers** [comment]
Is that for SciPy 1.3.0?

**Comment by QuLogic** [comment]
See Also parsing has become more strict. It must be actual references to other items and not just random text.

**Comment by QuLogic** [comment]
This should have been fixed by https://github.com/scipy/scipy/commit/efeca748f66b7ccbbb49770b9d011848a5337e17 in 1.3.0. I assume your build was 1.2.1 since Fedora has not updated due to blas troubles.

**Comment by larsoner** [comment]
I'll close since it sounds like the mystery has been solved -- ScyPy 1.2.0 should probably only be built with numpydoc 0.8 and SciPy 1.3+ can use numpydoc 0.9.1. But if this turns out not to be the case let's reopen
